esphome/components/light/light_color_values.h

```diff
@@ -86,16 +86,16 @@ class LightColorValues {
   static LightColorValues lerp(const LightColorValues &start, const LightColorValues &end, float completion) {
     LightColorValues v;
     v.set_color_mode(end.color_mode_);
-    v.set_state(esphome::lerp(completion, start.get_state(), end.get_state()));
-    v.set_brightness(esphome::lerp(completion, start.get_brightness(), end.get_brightness()));
-    v.set_color_brightness(esphome::lerp(completion, start.get_color_brightness(), end.get_color_brightness()));
-    v.set_red(esphome::lerp(completion, start.get_red(), end.get_red()));
-    v.set_green(esphome::lerp(completion, start.get_green(), end.get_green()));
-    v.set_blue(esphome::lerp(completion, start.get_blue(), end.get_blue()));
-    v.set_white(esphome::lerp(completion, start.get_white(), end.get_white()));
-    v.set_color_temperature(esphome::lerp(completion, start.get_color_temperature(), end.get_color_temperature()));
-    v.set_cold_white(esphome::lerp(completion, start.get_cold_white(), end.get_cold_white()));
-    v.set_warm_white(esphome::lerp(completion, start.get_warm_white(), end.get_warm_white()));
+    v.set_state(esphome::lerp(start.get_state(), end.get_state(), completion));
```

Comment:
Might as well directly use std
```suggestion
    v.set_state(std::lerp(start.get_state(), end.get_state(), completion));
```

Reply:
Done [`26a8882` (#9269)](https://github.com/esphome/esphome/pull/9269/commits/26a88827e5cf65f297df360ae5c1abbcf543c8b0)